### ISSUE TITLE
Demo page broken

### ISSUE THREAD

**Issue by letmaik** [consbio/Leaflet.ZoomBox#2]
Tested in latest Firefox.

**Comment by brendan-ward** [comment]
Hi @neothemachine, thanks for reporting.  Unfortunately, I am unable to reproduce the issue.  The demo page works fine for me on Firefox 39.0 (latest) on Windows.

What OS are you using?

Can you please define what is broken?  Does the page not load, tiles not load, or does the zoom tool not work as expected?

**Comment by letmaik** [comment]
Hi, sorry it appears it was caused by the HTTPS Everywhere browser 
extension which switches to https:

https://consbio.github.io/Leaflet.ZoomBox/

This doesn't work as leaflet's cdn has an invalid SSL certificate.

Am 07.07.2015 um 00:26 schrieb Brendan Ward:

**Comment by brendan-ward** [comment]
I fixed the Leaflet issue by pointing to their recommended CDN, and updated map tiles to OpenStreetMap  (the Stamen Design tiles failed over HTTPS too).

Everything appears to be playing nice over HTTPS now.

**Comment by nikmolnar** [comment]
Looks like the demo page is broken again.

**Comment by nikmolnar** [comment]
Looks like the basemap is also broken. Do you have a preferred alternative?

**Comment by brendan-ward** [comment]
Ah, I found it.  The problem is the link for the example, it is getting automatically set  to `https` by GitHub.  This is likely breaking most of our Leaflet.\* examples.

The basemap I found that supports HTTPs is OpenStreetMap: https://github.com/consbio/Leaflet.UTFGrid/blob/gh-pages/index.html#L59

I'm not the biggest fan of the styling.  So - should we fix the link to the examples so that they are only on HTTP, or should we migrate all examples to OpenStreetMap basemaps?
